ui/app/components/modals/new-account-modal.js

```diff
@@ -19,21 +20,31 @@ function mapDispatchToProps (dispatch) {
     hideModal: () => {
       dispatch(actions.hideModal())
     },
-    createAccount: () => {
-      dispatch(actions.addNewAccount())
-      dispatch(actions.hideModal())
+    createAccount: (identities, newAccountName) => {
+      dispatch(actions.addNewAccount(identities))
+        .then((newAccountAddress) => {
```

Comment:
Do we still need to save account label if user didn't enter a label?

Reply:
No. Fixed in 69682a3f